civictechprojects/static/css/partials/_SplashScreen.scss

```diff
@@ -0,0 +1,44 @@
+.SplashScreen-root {
+  background-color: $color-text-grey;
+  color: $color-text-light;
+  width: 100%;
+  height: 80vh;
+}
+
+.SplashScreen-content {
+  position: relative;
+  text-align: center;
+  transform: translateY(-50%);
+  top: 50%;
+}
+
+.SplashScreen-section {
+  padding-bottom: 15px;
+}
+
+.SplashScreen-find-projects-btn {
+  border: 1px solid $color-text-light;
+  color: $color-text-light;
+  background-color: transparent;
+  border-radius: 5px;
+  padding: 6px 12px;
+  transition: 300ms;
+  cursor: pointer;
+  min-width: 80px;
+  &:hover {
+    background-color: $color-orange;
+    color: $color-text-light;
+    border-color: $color-text-light;
+  }
+}
+
+.SplashScreen-create-project-btn {
+  border: 1px solid $color-orange;
+  color: $color-text-light;
+  background-color: $color-orange;
+  border-radius: 5px;
+  padding: 6px 12px;
+  margin: 0px 0px 0px 15px;
```

Comment:
And change this to `margin: 0px 0px 10px 15px;` to fix the other half. The key is that the buttons have the same margin-bottom value.